tests/test_local_move.py

```diff
@@ -181,3 +185,52 @@ def assert_correctness(local_move):
     # expect failure with ablated version of local move
     with pytest.raises(RuntimeError):
         assert_correctness(incorrect_local_move)
+
+
+def test_local_md_particle_density():
+    mol, _ = get_biphenyl()
+    ff = Forcefield.load_from_file("smirnoff_1_1_0_sc.py")
+
+    temperature = 300
+    dt = 1.5e-3
+    friction = 0.0
+    seed = 2022
+    cutoff = 1.2
+
+    # Have to minimize, else there can be clashes and the local moves will cause crashes
+    unbound_potentials, sys_params, masses, coords, box = get_solvent_phase_system(mol, ff, 0.0)
+
+    local_idxs = np.array([len(coords) - 1], dtype=np.int32)
```

Comment:
Is it worth parameterizing this test to run with different numbers of local_idxs?

Reply:
After further investigation, it doesn't seem like it. I switched to what seems like the more typical case, of the entire ligand.

Testing the case with random idxs is a bit more difficult, as a random index will be selected in C++, and doesn't seem to cover a different set of logic.